docker/Dockerfiles/Dockerfile-release

```diff
@@ -0,0 +1,42 @@
+FROM python:3.7.3-slim
+ARG RALLY_VERSION
+ARG RALLY_LICENSE
+
+ENV RALLY_RUNNING_IN_DOCKER True
+
+RUN apt-get -y update && \
+    apt-get install -y curl git gcc && \
+    apt-get -y upgrade && \
```

Comment:
Could we decrease the size of the image a bit further by cleaning up any downloaded files that `apt-get` has cached locally?

Reply:
As mentioned in https://docs.docker.com/develop/develop-images/#run (need to scroll further down to APT):

> Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.

The image we are basing on (python:3.7.3-slim) is based on the official debian stretch (https://github.com/docker-library/python/blob/4df129649329186fb42c9c33215b41d82bc0e58e/3.7/stretch/slim/Dockerfile#L7) so what we are doing here (i.e. `rm -rf /varl/ib/apt/lists/*`) should be enough.